playwright/tests/form-editor.test.ts

```diff
@@ -33,7 +33,7 @@ test.describe('Form Editor', () => {
   test('Edit input label', async () => {
     const input = editor.locatorFor('.block-input');
     await input.click();
-    const labelProperty = editor.locatorFor('[id=":re:-input"]');
+    const labelProperty = editor.locatorFor('#properties .ui-input').first();
```

Comment:
fyi @ivy-lmu: ":re:" is a random id by react, so not really a good idea to use this in a test case :)